quackosm/osm_extracts/geo2day.py

```
@@ -0,0 +1,177 @@
+        page_url (str): Page URL to load.
+        pbar (tqdm): Progress bar.
+
+    Returns:
+        list[Any]: List of geo2day region url objects for further processing.
+    """
+    from bs4 import BeautifulSoup
+
+    pbar.set_description_str(id_prefix)
+    region_objects = []
+
+    result = requests.get(page_url, headers={"User-Agent": _USER_AGENT})
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)